src/App.tsx

```diff
@@ -1,61 +1,154 @@
+import React, { useState } from 'react';
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+type User = {
+  id: number;
+  name: string;
+  email: string;
+};
+
+type Todo = {
+  id: number;
+  title: string;
+  user: User;
```

Comment:
The `user` property in the `Todo` type is expected to be of type `User`, but in the `todosFromServer` data, it seems to be referenced by `userId`. Ensure that the data structure aligns with this type definition.